lib/doctest.js

```diff
@@ -378,7 +388,15 @@ function rewrite$js(options, input) {
   //     produced by step 6 (substituting "step 6" for "step 2").
 
   function getComments(input) {
-    return (esprima.parse (input, {comment: true, loc: true})).comments;
+    return (esprima.parse (input, {
+      comment: true,
+      loc: true,
+      sourceType: options.sourceType
+    })).comments;
```

Comment:
Let's use a variable here and return `_.comments`.

Reply:
:zap: